understanding/20/audio-description-or-media-alternative-prerecorded.html

```diff
@@ -95,69 +95,41 @@ <h2>Benefits of Audio Description or Media Alternative (Prerecorded)</h2>
    
    <section id="examples">
       <h2>Examples of Audio Description or Media Alternative (Prerecorded)</h2>
-      
-      
-      <ul>
-         
-         <li>
-            
+
+      <dl>
+         <dt>A movie with audio description</dt>
+         <dd>
             <p>
-               								       
-               <strong>A movie with audio description.</strong>
-               							     
-            </p>
-            
-            <p> 
-               
                <strong>Describer: </strong>A title, "Teaching Evolution Case Studies. Bonnie Chen." A teacher shows photographs
                of birds with long, thin beaks.
             </p>
             
             <p> 
-               
                <strong>Bonnie Chen: </strong>"These photos were all taken at the Everglades."
             </p>
             
             <p> 
-               
                <strong>Describer: </strong>The teacher hands each student two flat, thin wooden sticks.
             </p>
             
             <p> 
-               
                <strong>Bonnie Chen: </strong>"Today you will pretend to be a species of wading bird that has a beak like this."
             </p>
             
             <p> 
-               
                <strong>Describer: </strong>The teacher holds two of the sticks to her mouth making the shape of a beak.
             </p>
             
-            <p>Transcript of audio based on the first few minutes of "
-               <a href="http://www.pbs.org/wgbh/evolution/educators/teachstuds/tvideos.html">Teaching Evolution Case Studies, Bonnie Chen</a>" (copyright WGBH and Clear Blue Sky Productions, Inc.)
-            </p>
-            
-         </li>
-         
-         <li>
-            
-            <p>
-               								       
-               <strong>An alternative for time-based media for a training video</strong>
-               							     
-            </p>
-            
-            <p>A company purchases a Training video for use by its employees and puts it on the companies
+            <p>Transcript of audio based on the first few minutes of "<a href="http://www.pbs.org/wgbh/evolution/educators/teachstuds/tvideos.html">Teaching Evolution Case Studies, Bonnie Chen</a>" (copyright WGBH and Clear Blue Sky Productions, Inc.)</p>
```

Comment:
unrelated to the changes, but would be good to use `https` here instead.

```suggestion
            <p>Transcript of audio based on the first few minutes of "<a href="https://www.pbs.org/wgbh/evolution/educators/teachstuds/tvideos.html">Teaching Evolution Case Studies, Bonnie Chen</a>" (copyright WGBH and Clear Blue Sky Productions, Inc.)</p>
```